### ISSUE TITLE
508-defect-3: Get your VA benefits link should be styled as a link, not a button

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45640]
## Point of contact

Josh Kim

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### User Story 

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

### Problem
The "Get your VA benefits" link is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click go to xyz _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link, and even more so when navigating by links/buttons using the rotor
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort.

<img width="1185" alt="Screen Shot 2022-08-12 at 11 33 39 AM" src="https://user-images.githubusercontent.com/14154792/184391365-7b6c7424-f839-49e7-8364-e3f756012c36.png">

## Reproduction steps

1. [Sign in as user 299 on the download VA benefit page](https://www.va.gov/records/download-va-letters/)
2. Right click the "Get your VA benefits" call to action and confirm you can open it in a new tab (it's a link) 

## Proposed solution or next steps

Use the action link component in the VA design system

## References, articles, or WCAG support

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [WCAG Success Criterion 3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by sortizsh** [comment]
We can fix this as part of DLD design.

**Comment by sortizsh** [comment]
We will make a recommendation to the Sitewide team to make this change, as part of the DLD designs.
